cmake/syclcc-launcher

```diff
@@ -66,7 +66,8 @@ if __name__ == '__main__':
   # When invoked with a command line for expected compiler (e.g. clang++), replace with a syclcc invocation.
   # Otherwise, e.g. for invocations of `ar` for linking static libraries, just continue with the command as-is.
   if os.path.samefile(cxx_compiler_exe, command_in):
-    command_line = [syclcc_exe, *syclcc_specific_args, *command_in_args]
+    command_line = syclcc_exe
+    command_line += ([*syclcc_specific_args, *command_in_args])
```

Comment:
Why is this change required?

Reply:
`syclcc_exe` now is an array, thus we'd have `[[python, syclcc], arg, arg2]` else.. but we could probably also use the `*` here.. :P